examples/qml_features/rust/src/containers.rs

```diff
@@ -139,7 +144,7 @@ impl ffi::RustContainersQt {
     fn insert_map(mut self: Pin<&mut Self>, key: QString, value: QVariant) {
         // SAFETY: map is a Q_PROPERTY so ensure we manually trigger changed
         unsafe {
```

Comment:
should the property mutable getter be unsafe still ? as you can bypass it with `rust_mut().property = ...`

Reply:
Or do we even need the property mutable getter ? Instead just have

- `property(&self) -> T`
- `set_property(self: Pin<&mut Self>, value: T)`
- `rust().property` (`&T`)
- `rust_mut().property` (`&mut T`)
- `property_changed(self: Pin<&mut Self>)`

Then if you go through `rust_mut()` it's up to the developer to call `property_changed` themselves.